src/index.js

```diff
@@ -105,6 +106,15 @@ DecafAppModule.config((appNameProvider, appAuthProvider, potionProvider, decafAP
             $window.localStorage.setItem('sessionJWT', process.env.GUEST_TOKEN);
         });
     }
+
+    // Track page state changes to Google Analytics
+    $transitions.onSuccess({}, function(transition) {
```

Comment:
Minor fixes:
```
$transitions.onSuccess({}, (transition) => {
    gtag('config', process.env.GA_TRACKING_CODE, {
        page_title: transition.to().data.title,
        page_location: $location.absUrl(),
        page_path: $location.path(),
    });
});
```
- Use arrow functions whenever applicable - almost every case
- Better use properties without quotes whenever possible (under scores are ok, dashes are not okay)
- We use trailing comma in object definitions - results in cleaner diffs
- Missing semicolon